CHANGELOG.md

```diff
@@ -3,7 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.0.1
+## dev
```

Comment:
Should this be updated to the latest release? Presumably since we are merging from dev into master this changelog shouldn't have anything only in dev

Reply:
We usually have dev as a suffix until the release PR